### ISSUE TITLE
[BUG] I'm not receiving an ICMP reply when attempting to ping from the UE to the internet.

### ISSUE THREAD

**Issue by Mouradnetworking** [HewlettPackard/PacketRusher#143]
**Describe the issue**

When attempting to ping Google's DNS (8.8.8.8) from the UE, only ICMP request packets are sent, but no ICMP reply is received. The following conditions have been checked:

Firewall (ufw): Inactive
iptables: No rules blocking ICMP traffic
IPv4 forwarding: 1

I'm using Packet Rusher for the radio part and Open5GS for the core part. 

**Screenshots**
![image](https://github.com/user-attachments/assets/cb48f1a5-5ddf-439f-b9e6-82af8fca02f6)

**Architecture :**
PRETTY_NAME="Ubuntu 22.04.4 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.4 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy

**Pcap:**
[ping.zip](https://github.com/user-attachments/files/17179522/ping.zip)

**Comment by linouxis9** [comment]
Hi @Mouradnetworking,

Nice to see you using PacketRusher. Have you made sure to setup NAT on the UPF's side as specified in Open5GS' documentation?

Thanks and cheers,
Valentin

**Comment by Mouradnetworking** [comment]
Hi Linou, thank you for your response.

I tried pinging the UPF (172.31.68.24), but it's not working.

Regarding NAT, I followed the Open5GS configuration using the commands:
""""
sudo iptables -t nat -A POSTROUTING -s 10.45.0.0/16 ! -o ogstun -j MASQUERADE
sudo ip6tables -t nat -A POSTROUTING -s 2001:db8:cafe::/48 ! -o ogstun -j MASQUERADE
""""
However, the issue persists.

Best regards,

**Comment by Mouradnetworking** [comment]
@linouxis9

**Comment by Mouradnetworking** [comment]
How about considering Docker instead? It might help us avoid these issues @linouxis9.

Kind regards,

**Comment by linouxis9** [comment]
Yes, it's also supported running UPF and PacketRusher on the same host using Docker.
You may look at this amazing project from @Borjis131 for that: https://github.com/Borjis131/docker-open5gs

**Comment by Mouradnetworking** [comment]
Thank you! I'll give it a try and get back to you.
